tidb-cloud/sql-proxy-account.md

```diff
@@ -0,0 +1,74 @@
+---
+title: SQL Proxy Account
+summary: Learn about the SQL proxy account in TiDB Cloud.
+---
+
+# SQL Proxy Account
```

Comment:
About SQL Proxy Accounts

Reply:
I would prefer to keep the topic title style consistent with that of other similar topics in TiDB docs. That is, omit the "About".